_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -100,10 +100,18 @@ This `input` element is not [focusable][] because of the `disabled` attribute.
 This `a` element is not [focusable][] because it moves focus to the `input` element whenever it receives focus.
 
 ```html
+<div role="dialog" aria-label="Sample Modal" aria-modal="true" style="border: solid black 1px; padding: 1rem;">
+    <label>Some input <input id="dialogFirst"></label><br />
+    <button>Close button</button>
+</div>
 <div aria-hidden="true">
-	<a href="/" style="position:absolute; top:-999em" onfocus="document.querySelector('input').focus()">First link</a>
+    <a href="#" id="sentinelAfter" style="position:absolute; top:-999em">Focus sentinel</a>
```

Comment:
I don't think we should leave the off screen style here. That's generally not what happens when a modal pops up. Could we instead turn down the opacity perhaps?

Reply:
Technically, the link does not need to be invisible. It only needs to be `aria-hidden` to trigger the rule 🤔

Reply:
@Jym77 The intent of this was to be more of a real-world dialog example. This is how we normally see dialog components built where they add this invisible, yet focusable sentinel that upon receiving focus wraps focus. I skipped the top sentinel in the interest of simplicity.

Reply:
While I understand the appeal of realism, I'm wondering if having the sentinel be visible after the dialog box and have the words like `When focus is placed here it will wrap to the dialog box` to make it even more obvious and self-documenting.

Reply:
Conclusion from TF call: needs more of an example description.